_includes/live-demos/drive-plugin-browse-controls/example.js

```diff
@@ -0,0 +1,28 @@
+tinymce.init({
+  selector: 'textarea#drive-browse-example',
+  height: 200,
+  menubar: false,
+  plugins: [
+    'tinydrive link image media'
+  ],
+  toolbar: 'custom | insertfile | link image media',
+  setup: function (editor) {
+    editor.ui.registry.addButton('custom', {
+      text: 'Custom browse',
+      onAction: function () {
+        var controls = editor.plugins.tinydrive.controls.pick({
+        }, function () {
```

Comment:
This reads very odd and makes it hard to understand what's going on, as it looks like the indentation is incorrect. Can you please look at changing this, either by adding a comment as to what the initial object is supposed to be or changing it so the empty object is on a single line if that doesn't need to be explained.